worlds/adventure/docs/setup_fr.md

```diff
@@ -0,0 +1,75 @@
+   **"NLua+KopiLua" jusqu'à ce que cette étape soit terminée.**
+- Sous Config > Personnaliser, cochez la case "Exécuter en arrière-plan". Cela empêchera la déconnexion du client pendant
+BizHawk s'exécute en arrière-plan.
+
+- Il est recommandé de fournir un chemin vers BizHawk dans votre host.yaml pour Adventure afin que le client puisse le démarrer automatiquement
+- En même temps, vous pouvez définir une option pour charger automatiquement le script connector_adventure.lua lors du lancement de BizHawk
+d'AdventureClient.
+Exemple d'installation Windows par défaut :
+```rom_args: "--lua=C:/ProgramData/Archipelago/data/lua/connector_adventure.lua"```
+
+## Configuration de votre fichier YAML
+
+### Qu'est-ce qu'un fichier YAML et pourquoi en ai-je besoin ?
+
+Votre fichier YAML contient un ensemble d'options de configuration qui fournissent au générateur des informations sur la façon dont il doit
+générer votre jeu. Chaque joueur d'un multimonde fournira son propre fichier YAML. Cette configuration permet à chaque joueur de profiter
+une expérience personnalisée à leur goût, et différents joueurs dans le même multimonde peuvent tous avoir des options différentes.
+
+### Où puis-je obtenir un fichier YAML ?
+
+Vous pouvez générer un yaml ou télécharger un modèle en visitant la [page des paramètres d'aventure](/games/Adventure/player-settings)
+
+### Quels sont les paramètres recommandés pour s'initier à la rando ?
+Régler la difficulté_switch_a et réduire la vitesse des dragons rend les dragons plus faciles à éviter. Ajouter Calice à
+local_items garantit que vous visiterez au moins un des châteaux intéressants, car il ne peut être placé que dans un château ou
+la salle des crédits.
+
+## Rejoindre une partie MultiWorld
+
+### Obtenez votre fichier de correctif Adventure
+
+Lorsque vous rejoignez un jeu multimonde, il vous sera demandé de fournir votre fichier YAML à l'hébergeur. Une fois cela fait,
+l'hébergeur vous fournira soit un lien pour télécharger votre fichier de données, soit un fichier zip contenant les données de chacun
+des dossiers. Votre fichier de données doit avoir une extension `.apadvn`.
+
+Faites glisser votre fichier de correctif vers AdventureClient.exe pour démarrer votre client et démarrer le processus de correctif ROM. Une fois le processus
+est terminé (cela peut prendre un certain temps), le client et l'émulateur seront démarrés automatiquement (si vous configurez l'émulateur
+chemin recommandé).
+
+### Connectez-vous au multiserveur
+
+Une fois le client et l'émulateur démarrés, vous devez les connecter. Dans l'émulateur, cliquez sur "Outils"
+menu et sélectionnez "Console Lua". Cliquez sur le bouton du dossier ou appuyez sur Ctrl+O pour ouvrir un script Lua.
+
+Accédez à votre dossier d'installation Archipelago et ouvrez `data/lua/ADVENTURE/adventure_connector.lua`, si ce n'est pas le cas
```

Comment:
Could you fix this line as well?  It should also be data/lua/connector_adventure.lua

Reply:
normally it's good if you want you can check